storage.go

```diff
@@ -115,6 +115,12 @@ const (
 			FROM report
 			WHERE reported_at < NOW() - $1::INTERVAL)`
 
+	deleteOrphanOCPRuleHits = `
```

Comment:
This is saying "delete everything in rule hit for org + cluster if there is no row in report table for given org + cluster".

I do understand the logic, but do we really need it?
What about the recommendation table? shouldn't we do the same "cleanup" there?

Reply:
> do we really need it?